app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -1851,6 +1838,19 @@ static boolean canHandleMimeType(@Nullable String mimeType) {
                 (mimeType.startsWith("image/") || mimeType.startsWith("video/") || mimeType.equals("text/plain")));
     }
 
+    private void onFetchInstanceSuccess(Instance instance) {
+        if (instance != null && instance.getMaxTootChars() != null) {
```

Comment:
Are you sure you need to check for `null`?

Reply:
No, I think I might be overcautious since I introduced bugs multiple times with `Call`/`Callback`

Reply:
But here it's Single and it would be null only if server intentionally would return it